crates/wasmedge-sys/src/async/fiber.rs

```diff
@@ -184,8 +184,16 @@ impl<'a> Future for TimeoutFiberFuture<'a> {
                 if libc::timer_create(libc::CLOCK_REALTIME, &mut sev, &mut timerid) < 0 {
                     return Poll::Ready(Err(()));
                 }
+
+                let timeout = self.deadline.duration_since(std::time::SystemTime::now());
+                if timeout.is_err() {
+                    return Poll::Ready(Err(()));
+                }
+                let timeout = timeout.unwrap().max(std::time::Duration::from_millis(100));
```

Comment:
Please refer to the following code to refine the code snippet for avoiding `unwrap`. Thanks!

```rust
let timeout = match self.deadline.duration_since(std::time::SystemTime::now()) {
    Ok(timeout) => timeout.max( ... ),
    Err(_) => return Poll::Ready(Err(()))
}
```